iroh-metrics/Cargo.toml

```diff
@@ -20,6 +20,7 @@ hyper = { version = "0.14.25", features = ["server", "client", "http1", "tcp"] }
 
 [dev-dependencies]
 tokio = { version = "1", features = ["io-util", "sync", "rt", "net", "fs", "macros", "time", "test-util"] }
+tokio-test = "0.4.2"
```

Comment:
is this actually used?

Reply:
Think so, the docs require a tokio runtime and this is one of the lighter/better options for docs from my google search when I initially did it.

Reply:
the docs don’t requrie tokio anymore no async in metrics